tests/base.py

```diff
@@ -47,6 +49,9 @@
 )
 test_app.testing = True
 
+DISCONNECTED_SIGNALS = {
+    contributor_added: [notify_added_contributor]
+}
```

Comment:
@sloria, `disconnect` requires the receiver function that you want to disconnect from the signal, so I used a dictionary instead